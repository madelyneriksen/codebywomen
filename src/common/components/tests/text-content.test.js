import React from 'react';
import renderer from 'react-test-renderer';
import TextContent from '../text-content.js';


describe("TextContent Component", () => {
  it("Matches Snapshot", () => {
    const html = '<p>Hello world!</p>'
    const tree = renderer.create(<TextContent html={html} />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})

