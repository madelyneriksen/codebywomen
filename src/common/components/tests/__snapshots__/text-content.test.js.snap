// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextContent Component Matches Snapshot 1`] = `
<div
  className="text-content"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<p>Hello world!</p>",
    }
  }
/>
`;
